src/lib/genesis_ledger/release_ledger.ml

```diff
@@ -1,2 +1,2 @@
 (* TODO: replace with release ledger *)
-include Testnet_posig_ledger
+include Test_ledger
```

Comment:
Can we have the `Release_ledger` have all the original contents of `Testnet_posig_ledger`?

Reply:
Done.